### ISSUE TITLE
VPS IONOS Linux XS ?

### ISSUE THREAD

**Issue by JmRy** [Ysurac/openmptcprouter#3054]
Bonjour tout le monde,

Quelqu'un aurait-il déjà fait une installation sur un vps ionos Linux XS 1vcode, 1Go ram, 10Go, 1.20€/TTC mois ?

Merci de votre retour d'expérience.

Cordialement

**Comment by cernoel** [comment]
it works on my side, just use debian and the install script.

**Comment by JmRy** [comment]
hi @cernoel 
How did you solve the IONOS default Firewall issue on the VPS? 
I opened port 65222, but I still can't connect. 

```
👤 Starting a new connection to: "5.xxx.xxx.xxx" port "65222"
⚙️ Starting address resolution of "5.xxx.xxx.xxx"
⚙️ Address resolution finished
⚙️ Connecting to "5.xxx.xxx.xxx" port "65222"
👤 Connection to "5.xxx.xxx.xxx" established
⚙️ Starting SSH session
😨 Connection to "5.xxx.xxx.xxx" closed with error: end of file
```

**Comment by cernoel** [comment]
i just disabled the firewall fully by removing all entries and add one entry allowing anything in, since the script installs shorewall anyway.

**Comment by cernoel** [comment]
maybe, i forgot to mention, i used latest stable on debian 11 server, no beta, 
```
Linux openmptcpr-1 5.4.207-mptcp #1 SMP Sun Jul 24 14:39:44 UTC 2022 x86_64
< OpenMPTCProuter VPS 0.1028 >
```

and on client i use proxmox with: https://download.openmptcprouter.com/release/v0.59.1/x86_64/targets/x86/64/openmptcprouter-v0.59.1-5.4-r0+16594-ce92de8c8c-x86-64-generic-ext4-combined.img.gz

**Comment by JmRy** [comment]
Hi @cernoel 

I have switched to Debian 10 and it is working. It suits me for the moment.
Thank you for your help.

**Comment by JmRy** [comment]
Hi @cernoel 

I have switched to Debian 10 and it is working. It suits me for the moment.

Thank you for your help.

**Comment by JmRy** [comment]
Hi all,
I encounter a new problem with this Ionos VPS.
The VPS geolocation is multiple. In Paris France, according to Ionos, but some sites are seen in Germany, others in London, it's hellish !!
Is there a function or add-on which allows you to freeze geolocation VPS?
Thanks for your answers

**Comment by cernoel** [comment]
as far as i can remember, you can tell ionos where to deploy you server when you buy a new one. maybe you have some obfuscation enabled in openmptcp router? (i saw some options there). i always use this for example to proxy via my server to different countries: https://ma.ttias.be/socks-proxy-linux-ssh-bypass-content-filters/

**Comment by cernoel** [comment]
btw. i've tested server: `6.1.0-13-cloud-amd64 0.1029-test`  with `openmptcprouter v0.59.1-5.4 r0+16594-ce92de8` on proxmox .. this also works quite well.

**Comment by JmRy** [comment]
> as far as i can remember, you can tell ionos where to deploy you server when you buy a new one. maybe you have some obfuscation enabled in openmptcp router? (i saw some options there). i always use this for example to proxy via my server to different countries: https://ma.ttias.be/socks-proxy-linux-ssh-bypass-content-filters/ ,maybe you can use this to test this, if there is really an issue with ionos?

That's one option, but it seems easier to me to just use a VPN service in that case.

So far, with my OVH VPS, I've been using NordVPN to get around blocks with Netflix, Prime Video, AppleID, and so on. With this IONOS VPS, it's not detected as a VPN, which is great for this services ! 

But its geolocation is variable, depending for the web site or service. It's located in Germany (for example, Google TV shows German content for movies details since I've been using this VPS). This morning, it was London... ?? I get the feeling that the IP class of this VPS isn't very well documented on the internet. 

Moreover, I lose VoWiFi as soon as I use it. 

These problems don't stem from my network configuration. I just have to switch back to my old OVH VPS, without any other changes, and everything goes back to normal.

In summary, the price is attractive € 12 including tax/year, but it is complicated to use it without hacks

In short, it's not very stable for professional use, not to mention that there's no IPV6 on this VPS.
